.gitignore

```diff
@@ -29,4 +30,7 @@ package.nls.*.json
 webviews-metafile.json
 # Ignore all l10n files except the default one (bundle.l10n.json)
 /localization/l10n/bundle.l10n.*.json
-.eslintcache
\ No newline at end of file
+.eslintcache
+.nyc_output
+/test/resources/screenshots
```

Comment:
maybe it should be test/artifacts. Since resource imply external files used as inputs in tests.